qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java

```diff
@@ -149,9 +155,94 @@ public void excludesSeveralIncorrectClassFromCheck() throws Exception {
         final Environment env = new Environment.Mock()
             .withFile("target/classes/Foo.class", bytecode)
             .withFile("target/classes/Bar.class", another)
-            .withExcludes("Foo,Bar")
+            .withExcludes("findbugs", "Foo,Bar")
             .withDefaultClasspath();
         new FindBugsValidator().validate(env);
     }
 
+    /**
+     * Test of findbugs-filter
+     * Having a class with the same name as interface (Remote)
+     * violates findbugs rule NM_SAME_SIMPLE_NAME_AS_INTERFACE
+     * so we disable the rule by providing the file.
+     * @throws Exception If something wrong
+     */
+    @Test
+    public void excludesFileFilter() throws Exception {
+        final File tmp = File.createTempFile("findbugs-filter", ".xml");
+        final Document document = DocumentHelper.createDocument();
+        final Element match = document
+            .addElement("FindBugsFilter")
+            .addElement("Match");
+        match.addElement("Bug").addAttribute(
+            "pattern",
+            "NM_SAME_SIMPLE_NAME_AS_INTERFACE"
+        );
+        match.addElement("Class").addAttribute(
+            "name",
+            "Remote"
+        );
+        new XMLWriter(
+            new FileOutputStream(tmp)
+        ).write(document);
+        final byte[] bytecode = new BytecodeMocker()
+            .withSource("class Remote implements java.rmi.Remote { }")
+            .mock();
+        final Environment env = new Environment.Mock()
+            .withFile("target/classes/Remote.class", bytecode)
+            .withDefaultClasspath()
+            .withExcludes(
+                "findbugs-filter",
```

Comment:
@ilyakharlamov Let's put `findbugs-filter` in a constant so we can try to avoid using `@SuppressWarnings("PMD.AvoidDuplicateLiterals")` at the start of the file

Reply:
@paulodamaso moved to constants